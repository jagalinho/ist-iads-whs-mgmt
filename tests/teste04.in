a 84f30a3a 1
r 84f30a3a
l
a 84f30a3a 96
l
r 84f30a3a
l
a 1234553a 1
m
a a234553a 2
m
r a234553a
m
l
x
