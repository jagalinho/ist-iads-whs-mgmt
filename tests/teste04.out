84f30a3a 96
1234553a 1
a234553a 2
1234553a 1
1234553a 1
1
